sidekick_plugin_installer/lib/src/plugin_context.dart

```diff
@@ -26,31 +25,51 @@ class PluginContext {
 
   static SidekickPackage? _sidekickPackage;
 
-  /// This is the plugin package that is getting installed from local source
+  /// Name of the plugin package to be installed
   ///
-  /// Plugin installer might need to know the local location during development,
-  /// to link the plugin dependency to a local path, instead of pub.dev where
-  /// the plugin is not yet published. (See [pubAddLocalDependency])
+  /// {@macro installer_parameter}
+  static String get name => env['SIDEKICK_PLUGIN_NAME']!;
+
+  /// Version constraint of the plugin package to be installed
   ///
-  /// Returns `null` when the plugin is not installed from local source, but
-  /// from a remote source (pub or git)
-  static DartPackage? get localPlugin {
```

Comment:
This makes it a breaking change. Do we want this breaking change? Keeping, or deprecating this getter is easy

Reply:
I deprecated it now